README.md

```diff
@@ -20,7 +20,39 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Start by require the gem:
+
+```ruby
+require 'void_logger'
+```
+
+Then you can use the logger:
+
+```ruby
+logger = VoidLogger.new
+logger.info "TEST" # nothing will be logged
+```
+
+You can also include the mixin in your module/class:
+
+```ruby
+require 'void_logger
+
+class Test
+  include VoidLogger::LoggerMixin
+end
+```
+
+This mixin will bring to your module/class two accessors:
+- logger: You can set the logger you want and will be return by default by the mixin
```

Comment:
The logger you can use in the class you mix VoidLogger::LoggerMixin into. Return the fallback logger (as defined above) unless you explicitly set a logger with the logger= method.